app/views/users/index.html.erb

```diff
@@ -7,9 +7,11 @@
   <ul>
     <% @users.each do |user| %>
       <li class='dflex user'>
-        <div class='dflex small-img'>
-          <img src="<%= user.photo %>">
-        </div>
+        <%= link_to user_path(user) do %>
+          <div class='dflex small-img'>
+            <img src="<%= user.photo %>">
+          </div>
+        <% end %>
         <div class='dflex flexcol user-name-nposts'>
           <a href="<%= user_path(user) %>">
             <div class='username'><%= user.name %></div>
```

Comment:
- [ ] Great job so far @EmiLoKiJu There are just a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Your `User Index` and `Show` pages look like the given wireframe great job with that :+1: But when I navigated to the `posts index` page, it didn't look like the given wireframe. The `user's name, number of posts, and image` weren't on that page as seen in the wireframe. So kindly make these changes so your design looks like the given wireframe :wink: 

| Your Design | The given wireframe |
|--------|--------|
| ![Screenshot from 2023-08-23 17-17-36](https://github.com/EmiLoKiJu/BlogApp/assets/102699958/44d091ee-0904-40eb-8c61-64eb6eb501c2) | ![image](https://github.com/EmiLoKiJu/BlogApp/assets/102699958/3139748b-87e6-4c44-9e66-255190540ae9) |